scene/gui/item_list.cpp

```diff
@@ -1294,6 +1309,9 @@ void ItemList::_bind_methods(){
 	ObjectTypeDB::bind_method(_MD("set_item_custom_bg_color","idx","custom_bg_color"),&ItemList::set_item_custom_bg_color);
 	ObjectTypeDB::bind_method(_MD("get_item_custom_bg_color","idx"),&ItemList::get_item_custom_bg_color);
 
+	ObjectTypeDB::bind_method(_MD("set_item_tooltip_enabled","idx","enable"),&ItemList::set_item_tooltip_enabled);
+	ObjectTypeDB::bind_method(_MD("get_item_tooltip_enabled","idx"),&ItemList::get_item_tooltip_enabled);
```

Comment:
It's not used consistently everywhere, but I think the convention is to use `is_` instead of `get_` for booleans. Like `set_item_tooltip_enabled/is_item_tooltip_enabled`.